eZ Platform/Admin UI/REQUIREMENTS.md

```diff
@@ -9,22 +9,26 @@ a mix of existing elements (in YUI) and rewritten elements (in a new framework),
 for the purpose of research and development (prototype), so that I can get a
 better picture of how such a hybrid approach may work."
 
+Underlying main goals are:
+- by adopting Symfony 3, we will provide developers, and especially the one who are experienced with eZ Publish or Symfony, substantially faster and easier way to develop, extend and customize eZ Platform.
+- by insuring an hybrid architecture that reuses existing elements relying on YUI, we will speed up availability of 2.x by not having to redevelop all features. Candidates for reuse are UDW, Sub-items view, Content view. If this is not possible, expectations and plans for 2.x will have to be be re-adjusted.
```

Comment:
See also: #12 Content Editing Prototype, a separate issue to focus on that. For the purpose of _this_ prototype, Symfony 3 Hybrid, we are mainly interested in what the Hybrid approach itself will look like, and need only do that which is necessary to achieve the desired experiment results.